### PR TITLE
Cookie compliance

### PR DESCRIPTION
**Pull request by apricot13**
Added cookies for bod, bucks and the generic theme

Without cookies maps and auto complete do not load. It degrades 'gracefully' - map is blank autocomplete does nothing.

image

### CODE REVIEW

File: src/contexts/googleContext.js [hunk 1]
```diff
@@ -1,25 +1,25 @@
...
+  const cookiesAccepted = checkCookiesAccepted()
+
   const { isLoaded } = useLoadScript({
     googleMapsApiKey: process.env.REACT_APP_GOOGLE_CLIENT_KEY,
     libraries: libs,
     preventGoogleFontsLoading: true,
   })
 
+  let value = {}
+  cookiesAccepted ? (value.isLoaded = isLoaded) : (value.isLoaded = false)
```
Comment (em-cd) [inline src/contexts/googleContext.js:19]: This looks a bit complicated, can we simplify it a bit? e.g.
```
<GoogleContext.Provider value={{ isLoaded: cookiesAccepted && isLoaded }} ...>
```

File: src/themes/generic/vars_generic.jsx [hunk 2]
```diff
@@ -14,6 +14,19 @@ export const vars_generic = {
   headerLogo: logo,
   mapSwitchSmall: true,
   usePresetTaxonomies: false,
+  cookieMessage: (
+    <>
+      <p>
+        We use cookies to collect informationabout how you use the
```
Comment (em-cd) [inline src/themes/generic/vars_generic.jsx:20]:
```suggestion
        We use cookies to collect information about how you use the
```

**Review by em-cd — CHANGES_REQUESTED** [review]
Just a couple of small changes then this is good to go on staging for testing! 🙏

**Comment by apricot13** [comment]
am merging since tests are failing regardless!

**Pull request merged**